Makefile

```
@@ -174,12 +174,23 @@ help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 ##@ Development
-
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) \
-		paths=./internal/controllers \
-		rbac:roleName=manager-role
+manifests: vendor controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./..." output:crd:artifacts:config=hack/crd/bases
```

not sure why just`crd` is included that way, maybe you meant:

```suggestion
	$(CONTROLLER_GEN) \ 
	        paths="./..." output:crd:artifacts:config=hack/crd/bases \
	        crd:crdVersions=v1 \
	        rbac:roleName=manager-role
```

also, should we have a new Makefile target, something like `make-clean-manifests` to run at the beginning of this to ensure we remove the previously generated manifests?

---

It is not only v1 we are using. It is already being overwritten on each `make manifests` makefile run.

---

That I know, I was thinking of having a clean repo before running it, however it is personal choice I guess